app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/optionsStepForm.jsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import CommentIcon from 'common/img/comment-inline.svg';
+import { DefectTypeItemML } from 'pages/inside/common/defectTypeItemML';
+import { useSelector } from 'react-redux';
+import { defectTypesSelector } from 'controllers/project';
+import Parser from 'html-react-parser';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import { ExecutionInfo } from 'pages/inside/logsPage/defectEditor/executionInfo';
+import { useIntl } from 'react-intl';
+import { messages } from './../../messages';
+import styles from './optionsStepForm.scss';
+
+const cx = classNames.bind(styles);
+
+export const OptionsStepForm = ({ accordionData, state, copyFromHistoryState }) => {
```

Comment:
it is redundantly have 2 props: `state` and `copyFromHistoryState` because you need only one of them for render
you can check using state in upper component before put it into props